manifests/importer.pp

```diff
@@ -20,16 +20,43 @@
   ################### 3. Telemetry dashboard ########################################
   # If using puppet_metrics_dashboard:
 
-    #   class { 'puppet_metrics_dashboard':
-      # add_dashboard_examples => true,
-      # overwrite_dashboards   => false,
-      # configure_telegraf     => true,
-      #enable_telegraf        => true,
-      #master_list            => $infranode,
-      #puppetdb_list          => [$pdb],
-      #postgres_host_list     => [$postgres],
-
-      #}
+  #
+  # this is where lists of master_list , puppetdb_list, postgres_host_list hosts are found using a puppetdb query
+  #
+
+  function puppet_enterprise::active_puppetdb_hosts() {
```

Comment:
Functions shouldn't be created inline, they should live under the `functions/` directory: https://puppet.com/docs/puppet/7.0/lang_write_functions_in_puppet.html

Reply:
So would it be like:

RSAN/manifests/functions/func1.pp
RSAN/manifests/functions/func2.pp

or

RSAN/functions/func1.pp
RSAN/functions/func2.pp

With each .pp file being only a single function?

Reply:
The latter, so `rsan/functions/func1.pp` would contain the definition for `rsan::func1`. It goes by the same naming convention that classes do and therefore only one function per file.

Reply:
Awesome, thanks Dylan!